src/rospkg/environment.py

```diff
@@ -109,7 +109,7 @@ def get_ros_package_path(env=None):
     """
     if env is None:
         env = os.environ
-    return env.get(ROS_PACKAGE_PATH, None)
+    return env.get(ROS_PACKAGE_PATH, '') + ':/usr/share'
```

Comment:
> I have the same patch for rospack waiting over here...

Regardless, it isn't required since the two versions of `rospack` can be installed side-by-side without consequence, but that is not true for `rospkg`.